### PR TITLE
Frecar/add image field

### LINKED ISSUE #272: Add field for image in Project
- [ ] Add field to model
- [ ] Add to queue object

### PR DESCRIPTION
**Pull request by frecar**
Fixes #272

### CODE REVIEW

**Comment by frecar** [comment]
Please merge: https://github.com/frigg/frigg-hq/pull/273 before this

**Comment by relekang** [comment]
The tests that use queue_object should be updated :)

File: frigg/builds/models.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class Project(TimeStampModel):
     queue_name = models.CharField(max_length=200, default=settings.FRIGG_WORKER_QUEUE)
     can_deploy = models.BooleanField(default=False, db_index=True)
     should_clone_with_ssh = models.BooleanField(default=False)
+    image = models.CharField(max_length=200, default="")
```
Comment (relekang) [inline frigg/builds/models.py]: Should this have `blank=True`? I think that it may be necessary for using django admin.

Comment (frecar) [inline frigg/builds/models.py]: You're right! :)

**Pull request merged**